### ISSUE TITLE
Is CRC available to ubuntu?

### ISSUE THREAD

**Issue by ManikantaMaddipati** [code-ready/crc#1123]
### General information

  * OS: Linux / macOS / Windows
  * Hypervisor: KVM / Hyper-V / hyperkit
  * Did you run `crc setup` before starting it (Yes/No)?

## CRC version
```bash
# Put the output of `crc version`
```
  
## CRC status
```bash
# Put the output of `crc status`
```

## CRC config
```bash
# Put the output of `crc config view`
```

## Host Operating System
```bash
# Put the output of `cat /etc/os-release` in case of Linux
# put the output of `sw_vers` in case of Mac
# Put the output of `systeminfo` in case of Windows
```

### Steps to reproduce

  1. 
  2. 
  3. 
  4. 

### Logs

You can start crc with `crc start --log-level debug` to collect logs.
Please consider posting this on http://gist.github.com/ and post the link in the issue.

**Comment by gbraad** [comment]
Yes, but manually steos might be needed. There is a pinned issue that explains this:

![image](https://user-images.githubusercontent.com/1894/77614272-ae9c1280-6f67-11ea-9985-ebd11fae693f.png)

Closing as a duplicate of https://github.com/code-ready/crc/issues/549

**Comment by ManikantaMaddipati** [comment]
Thanks
